pdfbrain/pdf_page.py

```diff
@@ -99,11 +99,12 @@ def rotation(self):
     @lazyproperty
     def label(self):
         '''Page label.'''
-        out = create_string_buffer(4096)
-        l = pdfium.FPDF_GetPageLabel(self._parent._ptr, self._pageno + 1, out, 4096)
-        if l < 4:
+        n_bytes = pdfium.FPDF_GetPageLabel(self._parent._ptr, self._pageno, None, 0)
+        out = create_string_buffer(n_bytes)
+        n_bytes = pdfium.FPDF_GetPageLabel(self._parent._ptr, self._pageno, out, n_bytes)
```

Comment:
If you don't like the double-call of `FPDF_GetPageLabel()`, this could be changed back to a static-size buffer to your preference.